libtransmission/bitfield.h

```diff
@@ -35,6 +39,11 @@ struct Bitfield
         this->setHasNone();
     }
 
+    /***
+    **** C++ Operators
+    ***/
+    Bitfield& operator=(const Bitfield&) = default;
```

Comment:
I'm wondering about having this line but not providing the others for rule of three/five/zero.

Plus, it's an `=default` so I'm wondering if what happens if this line is removed? If things still work, we should remove it

Reply:
Zero it is then.